databricks_cli/sdk/service.py

```diff
@@ -510,51 +529,105 @@ def put(self, path, contents=None, overwrite=None, headers=None):
         if overwrite is not None:
             _data['overwrite'] = overwrite
         return self.client.perform_query('POST', '/dbfs/put', data=_data, headers=headers)
-
+    
+    def put_test(self, path, contents=None, overwrite=None, headers=None):
+        _data = {}
+        if path is not None:
+            _data['path'] = path
+        if contents is not None:
+            _data['contents'] = contents
+        if overwrite is not None:
+            _data['overwrite'] = overwrite
+        return self.client.perform_query('POST', '/dbfs-testing/put', data=_data, headers=headers)
+    
     def mkdirs(self, path, headers=None):
         _data = {}
         if path is not None:
             _data['path'] = path
         return self.client.perform_query('POST', '/dbfs/mkdirs', data=_data, headers=headers)
-
+    
+    def mkdirs_test(self, path, headers=None):
+        _data = {}
+        if path is not None:
+            _data['path'] = path
+        return self.client.perform_query('POST', '/dbfs-testing/mkdirs', data=_data, headers=headers)
+    
     def move(self, source_path, destination_path, headers=None):
         _data = {}
         if source_path is not None:
             _data['source_path'] = source_path
         if destination_path is not None:
             _data['destination_path'] = destination_path
         return self.client.perform_query('POST', '/dbfs/move', data=_data, headers=headers)
-
+    
+    def move_test(self, source_path, destination_path, headers=None):
+        _data = {}
+        if source_path is not None:
+            _data['source_path'] = source_path
+        if destination_path is not None:
+            _data['destination_path'] = destination_path
+        return self.client.perform_query('POST', '/dbfs-testing/move', data=_data, headers=headers)
+    
     def delete(self, path, recursive=None, headers=None):
         _data = {}
         if path is not None:
             _data['path'] = path
         if recursive is not None:
             _data['recursive'] = recursive
         return self.client.perform_query('POST', '/dbfs/delete', data=_data, headers=headers)
-
+    
+    def delete_test(self, path, recursive=None, headers=None):
+        _data = {}
+        if path is not None:
+            _data['path'] = path
+        if recursive is not None:
+            _data['recursive'] = recursive
+        return self.client.perform_query('POST', '/dbfs-testing/delete', data=_data, headers=headers)
+    
     def create(self, path, overwrite=None, headers=None):
         _data = {}
         if path is not None:
             _data['path'] = path
         if overwrite is not None:
             _data['overwrite'] = overwrite
         return self.client.perform_query('POST', '/dbfs/create', data=_data, headers=headers)
-
+    
+    def create_test(self, path, overwrite=None, headers=None):
```

Comment:
Where did these set of functions come from?

Reply:
from autogeneration of this file - has nothing to do with my change. Seems like it was out of sync with changes made in universe.